cms/bundles/tests/test_viewsets.py

```diff
@@ -147,28 +164,86 @@ def test_bundle_edit_view__redirects_to_index_for_published_bundles(self):
         response = self.client.get(self.published_bundle_edit_url)
         self.assertRedirects(response, self.bundle_index_url)
 
-    def test_bundle_edit_view__updates_approved_fields_on_save_and_approve(self):
-        """Checks the fields are populated if the user clicks the 'Save and approve' button."""
+    def post_with_action_and_test(self, action: str, expected_status: BundleStatus, redirects_to: str | None = None):
```

Comment:
Nit-pick: Look like all function calls provide `redirects_to` so maybe we can simplify it?

Reply:
354bde99